src/hooks/useEngineConnectionSubscriptions.ts

```diff
@@ -193,6 +218,72 @@ export function useEngineConnectionSubscriptions() {
                 err(codeRef) ? [0, 0] : codeRef.range
               )
 
+              const getEdgeCutMeta = (): null | EdgeCutInfo => {
+                let chamferInfo: {
+                  segment: SegmentArtifact
+                  type: EdgeCutInfo['subType']
+                } | null = null
+                if (
+                  artifact?.type === 'edgeCut' &&
+                  artifact.subType === 'chamfer'
+                ) {
+                  const consumedArtifact = getArtifactOfTypes(
+                    {
+                      key: artifact.consumedEdgeId,
+                      types: ['segment', 'sweepEdge'],
+                    },
+                    engineCommandManager.artifactGraph
+                  )
+                  if (err(consumedArtifact)) return null
```

Comment:
We had this discussion on this exact same function I think. This is more of a type check than catching an error. If you like I can change the helper functions from artifact graph to return null instead of an error?

Reply:
Nah all good if it's effectively the same

Reply:
This is the pattern that we were considering whether we should make a custom lint against. The concern is that it's hiding bugs. It may be perfectly fine, though, like in cases like this.

So what should we do going forward? If we're accepting that this is okay, then we should remove the redundant `console.error()` in [trap.ts](https://github.com/KittyCAD/modeling-app/blob/6595fca000406a9aa29eaa8ea7c6f814b542a24b/src/lib/trap.ts#L19). Maybe just comment _why_ it's okay if it actually is, but no one is going to enforce that.